jest.config.js

```diff
@@ -0,0 +1,185 @@
+// For a detailed explanation regarding each configuration property, visit:
```

Comment:
Let's keep it simple

Reply:
this is the default generated config of jest :)
You mean you want to remove all of the comments?

Reply:
Yeah ;)